Admin/Extension/ChildExtension.php

```diff
@@ -3,12 +3,13 @@
 /*
  * This file is part of the Symfony CMF package.
  *
- * (c) 2011-2015 Symfony CMF
+ * (c) 2011-2016 Symfony CMF
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
 
+
```

Comment:
Looks like because there's a space missing at the start, or a fixer that normalised that spacing is turned off perhaps?